ledger/participant-integration-api/src/main/scala/platform/store/dao/events/Raw.scala

```diff
@@ -131,6 +141,8 @@ object Raw {
         PbFlatEvent(PbFlatEvent.Event.Created(event))
 
       override def witnesses: Seq[String] = raw.witnessParties
+
+      def stakeholders: Set[String] = raw.signatories.toSet ++ raw.observers
```

Comment:
why do you need this? The flat witnesses are the stakeholders.

Reply:
You may know that but I didn't and I would expect the next guy along would not know it either, this makes it explicit.